packages/driver/cypress/e2e/e2e/origin/user_agent_override.cy.ts

```diff
@@ -0,0 +1,50 @@
+// @see https://github.com/cypress-io/cypress/issues/22953
+// We modify the user agent to strip cypress and electron out of the user agent string to appear more chrome-like
```

Comment:
do we need to strip out `cypress`?  I thought it wasn't causing any issues and only `electron` was?  Or are we just stripping out all non-standard parts of the referrer?

Reply:
Just stripping out non standard parts of the user agent. Seems weird to keep it in there but currently there aren't ill adverse side effects with keeping it as far as we know